src/routes/blogs/deepseek-r1-on-device/+page.svx

```diff
@@ -0,0 +1,66 @@
+authorsLink: ['https://www.linkedin.com/in/parinitaparinita/', 'https://www.linkedin.com/in/sunghoon/', 'https://www.linkedin.com/in/kunal-v-16315b94/', 'https://www.linkedin.com/in/maanavdalal/']
+image: 'https://iili.io/2yV40bV.png'
+imageSquare: 'https://iili.io/2yV40bV.png'
+url: 'https://onnxruntime.ai/blogs/deepseek-r1-on-device'
+---
+
+Are you a developer looking to harness the power of your users' local compute for AI inferencing on PCs with NPUs, GPUs, and CPUs? Look no further! Expanding on the recent capability to run the models on new [Copilot+PCs on NPUs](https://blogs.windows.com/windowsdeveloper/2025/01/29/running-distilled-deepseek-r1-models-locally-on-copilot-pcs-powered-by-windows-copilot-runtime/), you now efficiently run these models on CPU and GPU devices as well. These models are available on Azure AI Foundry model catalog and [Hugging Face](https://huggingface.co/onnxruntime/DeepSeek-R1-Distill-ONNX).
+
+The DeepSeek ONNX models enables you to run DeepSeek on any GPU or CPU, achieving performance speeds 1.3 to 6.3 times faster than native PyTorch. These optimized models are coming soon via Azure AI Foundry and can be easily accessed via the command line or the [VS Code AI Toolkit](https://code.visualstudio.com/docs/intelligentapps/overview).
+
+VS Code AI Toolkit and included frameworks like Olive gives you a complete stack that enables you to convert, optimize, quantize and finetune these models. Olive provides ease of optimized and quantized model generation. ONNX Runtime Generative API gives you faster performance.
+
+## Convert, Quantize, and Run your models easily!
+
+Watch the video below and follow along on how you can covert the DeepSeek model to ONNX using Olive, and run it using a chat template on-device using ONNXRuntime Generative API.
+
+<!-- Video Embed -->
+<div>
+				<iframe
+					class="pb-2 w-full"
+                    height="600px"
+					src="https://www.youtube.com/embed/r_fXlTVHP_A"
+					title="YouTube video player"
+					frameborder="0"
+					allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+					allowfullscreen
+				/>
+			</div>
+
+The steps of how to acquire and play with the model are also shared [here](https://github.com/microsoft/onnxruntime/blob/gh-pages/docs/genai/tutorials/deepseek-python.md).
+
+## Easily Finetune your models with Olive.
+
+This [notebook](https://github.com/microsoft/Olive/blob/samuel100/deekseek-example/examples/getting_started/olive-deepseek-finetune.ipynb) provides a step-by-step guide to fine-tuning DeepSeek models using the Olive framework. It covers the process of setting up your environment, preparing your data, and leveraging Azure AI Foundry to optimize and deploy your models. The notebook is designed to help you get started quickly and efficiently with DeepSeek and Olive, making your AI development process smoother and more effective.
```

Comment:
Broken link. Update link to the main branch:

https://github.com/microsoft/Olive/blob/main/examples/getting_started/olive-deepseek-finetune.ipynb